meetings/2024/README.md

```diff
@@ -39,18 +29,35 @@ All schedule items must have a public issue or checked-in proposal that can be l
 
 ### Mon Apr 1, 2024
 
+- [Implementation specific documentation](https://github.com/dotnet/csharplang/issues/7898). (Bill, on a Monday)
```

Comment:
Can we move this to a different Monday?  Easter Monday is a holiday in the UK, and committee members may not be available.